src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberAnalysisResult.cs

```diff
@@ -412,5 +414,59 @@ public void ArgumentsWereCoerced()
             _argumentsCoerced = true;
 #endif
         }
+
+        internal MemberAnalysisResult WithoutReceiverArgument()
+        {
+            var badArguments = shift(BadArgumentsOpt);
+            var argsToParams = adjustArgsToParams(ArgsToParamsOpt);
+            var conversions = adjustConversions(ConversionsOpt);
+
+            return new MemberAnalysisResult(Kind, badArguments, argsToParams, conversions, BadParameter - 1, HasAnyRefOmittedArgument,
```

Comment:
>HasAnyRefOmittedArgument [](http://example.com/codeflow?start=109&length=24)

Can this be affected by receiver parameter?

Reply:
The only consumer of this property is overload resolution (there's a betterness rule), but this adjustment occurs after we're done with overload resolution.
I could put a dummy value or leave a comment if that helps.

Reply:
> I could put a dummy value or leave a comment if that helps

Consider leaving a comment.